src/pages/definePassengerType.tsx

```diff
@@ -265,7 +265,10 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): { props: De
     const passengerTypeAttribute = getSessionAttribute(ctx.req, PASSENGER_TYPE_ATTRIBUTE);
     const passengerTypeErrorsAttribute = getSessionAttribute(ctx.req, DEFINE_PASSENGER_TYPE_ERRORS_ATTRIBUTE);
 
-    if (!isPassengerType(passengerTypeAttribute) && (!ctx.query.groupPassengerType || !groupPassengerTypes)) {
+    if (
+        !passengerTypeAttribute ||
+        (!isPassengerType(passengerTypeAttribute) && (!ctx.query.groupPassengerType || !groupPassengerTypes))
```

Comment:
Does this check want to have just `!passengerTypeAttribute` rather than `!passengerTypeAttribute || !isPassengerType(passengerTypeAttribute`?